static/markdown/packages-main.md

```diff
@@ -0,0 +1,41 @@
+In Haskell, packages are configured and built with the Cabal package system built into GHC (and other compilers). For more specific details, see [The Cabal User Guide](https://www.haskell.org/cabal/users-guide/). The command line tools to download and install packages are either `cabal` or `stack`, each having different workflows. For details on their usage, see the documentation above.
+
+### Hackage
+
+Hackage is a repository of packages to which anyone can freely upload at any time. The packages are available immediately and documentation will be generated and hosted there. It can be used by cabal install.
+
+You can install a package using cabal by running:
+
+    $ cabal update
+    $ cabal install the-package
+
+Note that if you are not in a sandbox, this will install the package globally, which is often not what you want, so it is recommended to set up sandboxes in your project directory by running `cabal sandbox init`.
+
+[Go to Hackage →](https://hackage.haskell.org/packages/)
```

Comment:
While we're changing things, it would be better at this point to point to the main hackage url rather than the old packages page.

Reply:
Fixed here 0da454d